SpatialGDK/Documentation/content/get-started/dependencies.md

```diff
@@ -0,0 +1,27 @@
+# Get started: 1 - Get the Dependencies
+
+### Hardware
+* Refer to the <a href="https://docs.unrealengine.com/en-US/GettingStarted/RecommendedSpecifications" data-track-link="Clicked UE4 Recommendations|product=Docs|platform=Win" target="_blank">UE4 recommendation</a> (Unreal documentation)
```

Comment:
This is HTML which has affected the GitHub rendering.

Reply:
works fine with me